packages/rspeedy/plugin-react-alias/test/compat/legacy.test.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import type { URL } from 'node:url'
+
+import { createRsbuild } from '@rsbuild/core'
+import { describe, expect, test, vi } from 'vitest'
+
+import { LAYERS } from '@lynx-js/react-webpack-plugin'
+
+vi.mock('node:module', async (importOriginal) => {
+  const original = await importOriginal()
+  const { createRequire: originalCreateRequire } =
+    original as typeof import('node:module')
+  return {
+    ...original as object,
+    createRequire: vi.fn().mockImplementation((path: string | URL) => {
```

Comment:
Instead of mocking `createRequire` from `node:module`, can we set the `rootPath` option of `pluginReactAlias`? This could simplify the test.